cookbooks/allken_et_al_2012_rift_interaction/doc/allken.md

```diff
@@ -0,0 +1,52 @@
+(sec:cookbooks:allken)=
+# Rift interaction in brittle-ductile coupled systems
+
+*This section was contributed by Cedric Thieulot*
+
+The setup for this experiment is based of {cite:t}`alht11,alht12,alhf13`.
+The Cartesian domain is $L_x\times L_y\times L_z=210\text{ km}\times 210\text{ km}\times 30\text{ km}$
+in size (see {numref}`fig:allken_setup`).
+It is a simple 2-layer crustal model consisting of $h_{uc}=15\text{ km}$ of upper crust (characterised by a Mohr-Coulomb rheology with cohesion
+$C^0=20\text{ MPa}$ and angle of friction $\phi^0=16^o$) and $h_{lc}=15\text{ km}$ of lower crust
+(characterised by a Newtonian viscous rheology with a viscosity $\eta_{lc}=10^{20}\text{ Pa.s}$).
+The flow is assumed to be incompressible and isothermal.
+Both materials have the same density $\rho=2800\text{ kg}/\text{m}^3$ and the gravity is vertical with magnitude $9.81\text{ m}/\text{s}^2$.
+
+```{figure-md} fig:allken-setup
+<img src="allken_setup.png" style="width:80.0%" />
+
+(a) Model setup showing box of dimension. (b) Rheological profile implemented: Mohr-Coulomb plasticity in the upper crust and a fixed viscosity. (c) Frictional plastic strain weakening behavior of the upper crust. Figure taken from {cite}`alht12`.
```

Comment:
Model setup showing box of dimension....
Something missing from this sentence?

Reply:
corrected